modules/web/src/app/settings/admin/defaults/component.ts

```diff
@@ -156,13 +156,29 @@ export class DefaultsComponent implements OnInit, OnDestroy {
     this.onSettingsChange();
   }
 
+  onStaticLabelsChange(labels: StaticLabel[]): void {
+    if (this._checkLabels(labels)) {
+      this.settings.staticLabels = labels;
+      this.onSettingsChange();
+    }
+  }
+
   isLastEnabledOS(os: OperatingSystem): boolean {
     if (this.allowedOperatingSystems.length === 1) {
       return os === this.allowedOperatingSystems[0];
     }
     return false;
   }
 
+  private _checkLabels(staticLabels: StaticLabel[]): boolean {
+    let correct = true;
+    staticLabels.forEach(label => {
+      if (!label.key?.length || !label.values?.length) {
+        correct = false;
+      }
+    });
+    return correct;
```

Comment:
Could be simplified to:
```suggestion
    return staticLabels.every(label => label.key?.length &&  label.values?.length));
```